backend/de.metas.acct.base/src/main/java-legacy/org/compiere/acct/Doc_InOut.java

```diff
@@ -374,6 +375,7 @@ private Fact createFacts_PurchasingReceiptLine(
 				.setAmt(null, roundToStdPrecision(costs))
 				.setQty(line.getQty().negate()) // (-) Qty
 				.bpartnerId(line.getBPartnerId(costElement.getId()))
+				.bpartnerLocationId(line.getBPartnerLocationId(costElement.getId()))
```

Comment:
wondering if it would be better to have a method called `bpartnerAndLocationId` just to make the intention clear and without hidden side effects. wdyt?